extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -1,13 +1,10 @@
 import React, { useEffect, useState } from "react";
 import { useLocation } from "react-router-dom";
 import { useDispatch, useSelector } from "react-redux";
+import { Button } from "@stellar/design-system";
```

Comment:
My overall strategy is to:

1) look at the Figma doc and try to figure out what elements are needed (buttons, boxes, heeadlines, etc)
2) Look through SDS and see if any of these things exist already. In this case, we can use the SDS `Button` components for the buttons at the bottom
3) For everything else, I'm creating custom styled elements:

For example, `TransactionHeader` (https://github.com/stellar/freighter/pull/347/files#diff-c8138d71833ff8dd4d8236105f1d6eb8b37c160861e85762f261af6c31b8d8cd)